js/main.js

```diff
@@ -87,48 +90,93 @@ Site.Camera = {
         duration: 100,
         easing: 'linear',
         step: function(val) {
-          _this.$fader.val(val);
+          _this.$xfader.val(val);
           _this.setStyleFromFader(val);
         }
     });
   },
 
-  bindFader: function() {
-    var _this = this,
-      faderValue;
+  bind: function() {
+    var _this = this;
 
-    _this.$fader.on('input', function() {
-      faderValue = $(this).val();
-      Cookies.set('faderValue', faderValue);
+    // Bind crossfader
+    _this.$xfader.on('input', function() {
+      var faderValue = $(this).val();
       _this.setStyleFromFader(faderValue);
     });
 
+
+    // Bind zoomfader
+    _this.$zoomfader.on('input', function() {
+      var faderValue = parseInt($(this).val());
+      _this.setZoom(faderValue);
+    });
+
+    // Bind pointer position
+    $(document).mousemove(function(e) {
+      var valX = e.pageX / window.innerWidth;
+      var valY = (e.pageY - window.pageYOffset) / window.innerHeight;
+
+      _this.setLocation(valX,valY);
+
+    });
+
+    // Bind keyboard shortcuts
     $(document).keydown(function(event) {
       if (event.keyCode >= 49 && event.keyCode <= 57) {
-        faderValue = ((event.keyCode - 48) * 10);
-        _this.$fader.val(faderValue);
-        Cookies.set('faderValue', faderValue);
+        var faderValue = ((event.keyCode - 48) * 10);
+        _this.$xfader.val(faderValue);
         _this.setStyleFromFader(faderValue);
       }
     });
   },
 
+  setLocation: function(x,y) {
+    var _this = this;
+
+    if(_this.zoomed && !_this.$zoomfader.is(':focus')) {
```

Comment:
this is not focus condition isn't working. When zooming in the fader the position freaks out and bounces around